pinot-spi/src/main/java/org/apache/pinot/spi/data/DateTimeFormatSpec.java

```diff
@@ -97,6 +157,19 @@ public DateTimeFormatSpec(int columnSize, String columnUnit, String columnTimeFo
     _patternSpec = new DateTimeFormatPatternSpec(columnTimeFormat, sdfPattern);
   }
 
+  public DateTimeFormatSpec(String timeFormat, String columnTimeFormat, @Nullable String timeZone) {
```

Comment:
Same here, let's not add new constructors for this class

Reply:
Removed in the latest commit